test/SILGen/keypaths.swift

```diff
@@ -338,3 +342,15 @@ func subscripts<T: Hashable, U: Hashable>(x: T, y: U, s: String) {
   _ = \Subscripts<T>.[Bass()]
   _ = \Subscripts<T>.[Treble()]
 }
+
+// CHECK-LABEL: sil hidden @{{.*}}subclass_generics
+func subclass_generics<T: C<Int>>(_: T) {
```

Comment:
Might be worth testing something like `T : C<U>, U` also

Reply:
Also please test cases where the member is in an extension of your class

Reply:
I think I already have some extension tests.